src/core/utils/slaac_address.cpp

```diff
@@ -87,7 +88,8 @@ void Slaac::UpdateAddresses(otInstance *    aInstance,
 
         if (!found)
         {
-            otIp6RemoveUnicastAddress(aInstance, &address->mAddress);
+            static_cast<Instance *>(aInstance)->GetThreadNetif().RemoveUnicastAddress(
```

Comment:
Does this make any real functional change? If we are looking towards making updates around how SLAAC is implemented, let's combine all those changes in a separate PR.

Reply:
Yes, `otIp6RemoveUnicastAddress()` adds or removes the interface in the pool, and this change adds or removes it as native address just like mesh local address.

Reply:
Are we planning to remove `otIp6SlaacUpdate()` API?

Reply:
I hope to do so. However, since this PR and this change is required by LwIP(SLAAC and DHCP changes being notified through this callback), I tend to submit separate PR removing the API, just like #3287 .

Reply:
Sounds good, I think this will look cleaner once we remove the API.